localstack/services/awslambda/lambda_api.py

```diff
@@ -312,14 +312,14 @@ def process_apigateway_invocation(func_arn, path, payload, stage, api_id, header
         LOG.warning('Unable to run Lambda function on API Gateway message: %s %s' % (e, traceback.format_exc()))
 
 
-def construct_invocation_event(method, path, headers, data, query_string_params={}):
+def construct_invocation_event(method, path, headers, data, query_string_params={}, isBase64Encoded=False):
```

Comment:
nitpick: Let's better use `snake_case` instead of `camelCase` (i.e., `is_base64_encoded`).